zulipterminal/model.py

```diff
@@ -1086,6 +1089,13 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
             set_count(list(message_ids_to_mark & indexed_message_ids),
                       self.controller, -1)
 
+    def _handle_update_emoji_event(self, event: Event) -> None:
+        """
+        Handle update of emoji
+        """
+        realm_emojis = event['realm_emoji']
+        self.active_emoji_data.update(realm_emojis)
```

Comment:
As I mentioned previously, events are not in the same format as the original data, so a dict update is not feasible here.